src/front-end/index.html

```diff
@@ -65,19 +67,28 @@
   </div>
 
   <button id="pay-button" disabled="">
+    <!-- <span id="refresh-icon"><img src="./images/refresh_white.png" alt="refresh"></span> -->
     <div id="button-message">Pay Now</div>
     <div id="button-dots" class="dots white-dots">
       <span></span>
       <span></span>
       <span></span>
     </div>
+
+    <div class="checkmark draw"></div>
+
+    <!-- Cross -->
+    <svg class="container container--fail" viewBox="0 0 50 50">
+      <path class="container__path cross" fill="none" d="M16 16 34 34 M34 16 16 34"></path>
+    </svg>
+    <script>$(".container").toggle();</script>
```

Comment:
don't write js here, write it in the js file

Reply:
Moved this in the latest change